views/templates/admin/tabs/help.tpl

```diff
@@ -0,0 +1,100 @@
+                    </div>
+                </div>
+                <div>
+                    <div class="numberCircle">2</div>
+                    <div class="numberCircleText">
+                        <p class="numberCircleText">
+                            {l s='switch to offline mode and allow your clinkts to send you an email directly through the window' mod='psreassurance'}
+                        </p>
+                    </div>
+                </div>
+                <div>
+                    <div class="numberCircle">3</div>
+                    <div class="numberCircleText">
+                        <p class="numberCircleText">
+                            {l s='customize your chat window according to the colors of your store' mod='psreassurance'}
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="helpContentRight">
+            <div class="helpContentRight-sub">
+                <b>{l s='Need help ?' mod='psreassurance'}</b><br>
+                {l s='Find here the documentation of this module' mod='psreassurance'}
+                <a class="btn btn-primary" href="{$guide_link|escape:'htmlall':'UTF-8'}" target="_blank" style="margin-left:20px;" href="#">
+                    <i class="fa fa-book"></i> {l s='Documentation' mod='psreassurance'}</a>
+                </a>
+                <br><br>
+                <div class="tab-pane panel" id="faq">
+                    <div class="panel-heading"><i class="icon-question"></i> {l s='FAQ' mod='psreassurance'}</div>
+                    {if $apifaq}
+                        {foreach from=$apifaq item=categorie name='faq'}
+                            <span class="faq-h1">{$categorie->title|escape:'htmlall':'UTF-8'}</span>
+                            <ul>
+                                {foreach from=$categorie->blocks item=QandA}
+                                    {if !empty($QandA->question)}
+                                        <li>
+                                            <span class="faq-h2"><i class="icon-info-circle"></i> {$QandA->question|escape:'htmlall':'UTF-8'}</span>
+                                            <p class="faq-text hide">
+                                                {$QandA->answer|escape:'htmlall':'UTF-8'|replace:"\n":"<br />"}
+                                            </p>
+                                        </li>
+                                    {/if}
+                                {/foreach}
+                            </ul>
+                            {if !$smarty.foreach.faq.last}<hr/>{/if}
+                        {/foreach}
+                    {/if}
+                </div>
+                {l s='You couldn\'t find any answer to your question ?' mod='psreassurance'}
+                <b><a href="http://addons.prestashop.com/contact-form.php" target="_blank">{l s='Contact us on PrestaShop Addons' mod='psreassurance'}</a></b>
```

Comment:
This is going to be a native module, right ? So it will only appear on github, not Addons.

Reply:
there is no help tab